source/jormungandr/jormungandr/scenarios/helpers.py

```diff
@@ -30,6 +30,10 @@
 from __future__ import absolute_import, print_function, unicode_literals, division
 from navitiacommon import response_pb2
 
+AIR_POLLUTANTS_ESTIMATION_NOX_COEFF = 0.44
+AIR_POLLUTANTS_ESTIMATION_PM10_COEFF = 0.056
```

Comment:
Can we put comments about where these values come from?

Reply:
There is no official resource. I took it from the ticket as proposed.

Reply:
There is some details in https://navitia.atlassian.net/browse/NAV-1760?focusedCommentId=39713. We could make a link to official documents and explain how we kept this average value. Maybe @wassimbenaissa can help doing that?

Reply:
Done